testdriver/acceptance/exec-output.yaml

```
@@ -18,9 +18,7 @@ steps:
         output: expectedDate
         code: |
           $date = (Get-Date).AddMonths(1)
-          Write-Output $date.ToString("ddd MMM d yyyy")
-      - command: assert
-        expect: ${OUTPUT.expectedDate} is not null or empty
+          Write-Output $date.ToString("MMMM dd, yyyy")
```

Putting it back to how it was:
```suggestion
          Write-Output $date.ToString("ddd MMM d yyyy")
```

---

oops sorry!!